assets-src/js/multiselect-module.js

```diff
@@ -518,12 +533,11 @@ MultiselectButtons.prototype.selectOption = function (option) {
     buttonEl.className = 'remove-option';
     buttonEl.type = 'button';
     buttonEl.id = `${this.idBase}-remove-${index}`;
-    buttonEl.setAttribute('aria-describedby', `${this.idBase}-remove`);
     buttonEl.dataset.value = selected.value;
     buttonEl.addEventListener('click', () => {
         this.removeOption(option);
     });
-    buttonEl.innerHTML = selected.text + ' ';
+    buttonEl.innerHTML = `<span class="sr-only">Remove </span> ${selected.text} `;
```

Comment:
The class exists in [_header.scss](https://github.com/w3c/w3c-website-templates-bundle/blob/main/assets-src/styles/sass/60-advanced-components/_header.scss#L77) and ends up in the compiled stylesheet. It also extends the `visuallyhidden` class

Reply:
@NicolaSaunders can you describe the difference between the classes `visuallyhidden` and `sr-only` and which one should be used in the case above? It's not clear to us when we should apply those.
Thanks

Reply:
Side note the `visuallyhidden` class is defined in the deign system https://design-system.w3.org/styles/how-to-hide-and-show-things.html#visually-hiding-items but the `sr-only` class isn't.

@NicolaSaunders if `sr-only` is useful, it should be added to the design system.

Reply:
@deniak @vivienlacourba @jean-gui The `sr-only` class extends the default styles of the `visuallyhidden` class, but differs in that at the specified media breakpoint the content that it was hiding is made visible (by adopting the opposite to the styles used in the `visuallyhidden` class). It is entirely specific to the account button in the header, hence why it is not documented as a helper class in the design system.

Apologies for the confusion. Naming things can be hard at times and this could probably be improved!

Reply:
Thanks for the details. So for that PR, @jean-gui's right and the right class to apply is `visuallyhidden`.